### PR TITLE
Bugfix/#2416 on Create news upload image

### PR DESCRIPTION
**Pull request by KirillAbramo**
Bug was fixed, Cancel and Submit buttons are available and working: buttons block removed from the NgIf statement in HTML, CSS file corrected, isCropper -  set to true to make browse option works correctly.
https://github.com/ita-social-projects/GreenCity/issues/2416
image

image

### CODE REVIEW

**Comment by idzhersh** [comment]
Screenshot (350)
Is this 'console.log' necessary?

**Comment by KirillAbramo** [comment]
> src/app/component/shared/components/drag-and-drop/drag-and-drop.component.ts
> 1Screenshot (350)
> Is this 'console.log' necessary?

log removed

**Review by paslavskyi9roman — APPROVED** [review]

File: src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss [hunk 1]
```diff
@@ -57,22 +57,20 @@ img {
...
-      width: 154px;
-      height: 48px;
-      color: var(--primary-green);
-      font-weight: bold;
-    }
+  button {
+    margin-left: 20px;
+    border: 1px solid var(--primary-green);
+    border-radius: 3px;
+    background-color: #fff;
```
Comment (yuriiLisunov) [inline src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss:69]: maybe here would be better put color to some variable?

File: GreenCityClient [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit cc5e83cc7c71d76324ec074c23101c200aa686ef
```
[the remark this answers is not in the document]
Comment (KirillAbramo) [reply GreenCityClient]: yes, changed it, no json in a commit now

File: src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss (hunk 1 above)
Comment (KirillAbramo) [reply src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss]: added var for white color

**Review by idzhersh — APPROVED** [review]

**Review by VitalyBorysenko — APPROVED** [review]

**Review by AlekOK — APPROVED** [review]

**Review by yuriiLisunov — APPROVED** [review]

**Pull request merged**